src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -296,11 +321,12 @@ const ReleaseContent = () => {
           release={release}
           renderAllFilesButton={
             <Button
-              className="govuk-!-width-full"
+              className="govuk-!-width-full govuk-!-margin-bottom-0"
```

Comment:
Might be a good idea to add some default cascading styling to `ReleaseDataAndFilesAccordion` so that the renders buttons are automatically full width and have no bottom margin to avoid repeating ourselves all over the place e.g:

```scss
.buttonContainer {
  :global(.govuk-button) {
    margin-bottom: 0;
    width: 100%;
  }
}
```

And in markup:

```tsx
<div className={styles.buttonContainer}>
  {renderAllFilesButton}
</div>
```